src/pkg/robot/model/robot.go

```diff
@@ -67,6 +68,9 @@ func (rq *RobotCreate) Valid(v *validation.Validation) {
 	if utils.IsContainIllegalChar(rq.Name, []string{",", "~", "#", "$", "%"}) {
 		v.SetError("name", "robot name contains illegal characters")
 	}
+	if rq.ExpiresAt < 0 {
+		v.SetError("expires_at", "robot account must set a expiration time.")
```

Comment:
Incorrect error message, and the API caller can leave this attribute unset.

Reply:
updated.